app/models/sign_in/session_container.rb

```diff
@@ -22,7 +22,7 @@ class SessionContainer
       presence: true
     )
 
-    def initialize(session:, # rubocop:disable Metrics/ParameterLists
```

Comment:
Is this needed?

Reply:
I think this is because the rubocop rule was changed for some reason for parameter list. Instead of changing the rule itself, I would refactor or disable the rubocop rule in the specific place it's affecting your code

Reply:
Yes, I see that now.

Reply:
Please add this line back